### ISSUE TITLE
Nullable boolean fields in PostgreSQL cause an error in Grafana 8

### ISSUE THREAD

**Issue by ariofrio** [grafana/grafana#37462]
**What happened**: When I upgraded to Grafana 8, some of my queries started failing with the following error:

```
convert frame from rows error: sql: Scan error on column index 11, name "online": sql/driver: couldn't convert <nil> (<nil>) into type bool
```

**What you expected to happen**: I expected the queries to continue to work, or at least to have a straightforward workaround.

**How to reproduce it (as minimally and precisely as possible)**: Create a PostgreSQL data source in a Grafana 8 instance. Create a data panel, select the PostgreSQL data source, click Edit SQL and type in `select now() as time, null::bool`. Select the Table visualization (or any visualization). Choosing to format as Table or Time Series doesn't make a difference either. Instead of showing the null value in the table, an error appears:

```
convert frame from rows error: sql: Scan error on column index 0, name "bool": sql/driver: couldn't convert <nil> (<nil>) into type bool
```

Change the SQL to `select now() as time, null::int` and now it works, representing the null value as an empty cell in the Table visualization.

**Anything else we need to know?**: I can convert the nullable boolean columns to strings in the query for the purposes of visualization, although this would be tedious since there are 20+ columns and I'm currently using a wildcard (`select * from ...`) for forward compatibility. This used to work in Grafana 7. I assume it's related to [changing the underlying data structure to data frames](https://grafana.com/docs/grafana/latest/installation/upgrading/#postgres-mysql-microsoft-sql-server-data-sources) (https://github.com/grafana/grafana/issues/35534) for SQL data sources. From the [data frames documentation](https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/), it's not clear to me that nullable booleans would not be supported (but nullable ints, texts, etc. are). This leads me to think this is an unintended bug.

**Environment**:
- SDK version: 
- Grafana version: v8.0.6 (68fe9e3431)
- Go version: 

I previously reported this as https://github.com/grafana/grafana-plugin-sdk-go/issues/386. Not sure where is the right place to report Grafana bugs such as this one.

**Comment by ying-jeanne** [comment]
We need to support nullable boolean, I think this should be fixed in sdk in order to be fixed for all SQLs.

**Comment by ying-jeanne** [comment]
Hey @ariofrio,

This bug has already been fixed in Grafana v8.1.0, could you please take a look and let us know if there is still issues.

Thanks,

**Comment by ying-jeanne** [comment]
fix in sdk with verison 111 integrated on branch 8.1
